.travis.yml

```diff
@@ -17,13 +17,8 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then source script/travis/install_cuda.sh; fi
 
 script:
-  - cd test/performance_tests
-  - make
-  - cd ../unit_tests
-  - make
+  - mkdir build && cd build/
+  - cmake ../ && cmake --build . && ctest --output-on-failure
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cd cuda; fi
```

Comment:
This command fails on Linux because it expects that current location is `unit_tests` direction which is not.